bin/ampcombi_download.py

```diff
@@ -1,78 +1,144 @@
 #!/usr/bin/env python3
 
 #########################################
-# Authors: [Anan Ibrahim](https://github.com/brianjohnhaas), [Louisa Perelo](https://github.com/louperelo)
+# Authors: [Anan Ibrahim](https://github.com/Darcy220606/AMPcombi), [Louisa Perelo](https://github.com/louperelo)
 # File: amp_database.py
 # Source: https://github.com/Darcy220606/AMPcombi/blob/main/ampcombi/amp_database.py
-# Source+commit: https://github.com/Darcy220606/AMPcombi/commit/a75bc00c32ecf873a133b18cf01f172ad9cf0d2d/ampcombi/amp_database.py
-# Download Date: 2023-03-08, commit: a75bc00c
 # This source code is licensed under the MIT license
 #########################################
 
-# TITLE: Download the DRAMP database if input db empty AND and make database compatible for diamond
+# TITLE: Download the reference database specified by the user.
```

Comment:
Shouldn't this script be wrapped up within AMPcombi itself, e.g. `ampcombi download ...`?

Reply:
It is but u remember the issue with it downloading the ref. database every time for each sample? We did that to only let it run once in the pipeline

Reply:
Also as i wrote in the PR comment, for this PR to run as it should we still need to update the module to v2.0.1, which has a sep. PR open in modules ;)